pints/_evaluation.py

```diff
@@ -413,6 +413,43 @@ def clear(q):
         return errors
 
 
+class MultiSequentialEvaluator(Evaluator):
+    """
+    Evaluates a list of functions (or callable objects) for a list of input
+    values of the same length, and returns a list containing the calculated
+    function evaluations.
+
+    Extends :class:`Evaluator`.
+
+    Parameters
+    ----------
+    functions : list of callable
+        The functions to evaluate.
+    args : sequence
+        An optional tuple containing extra arguments to each element in
+        functions, ``f``. If ``args`` is specified, ``f`` will be called as
+        ``f(x, *args)``.
+    """
+    def __init__(self, functions, args=None):
+        super(MultiSequentialEvaluator, self).__init__(functions[0], args)
+
+        # Check functions
+        for function in functions:
+            if not callable(function):
+                raise ValueError('The given function must be callable.')
+        self._functions = functions
+        self._num_functions = len(functions)
+
+    def _evaluate(self, positions):
+        if len(positions) != self._num_functions:
+            raise ValueError('Number of positions does not equal number of '
+                             'functions.')
+        scores = [0] * len(positions)
```

Comment:
I suppose this could even be a
```
return [f(x, *self._args) for f, x in zip(self._functions, positions)]
```
?

Reply:
(I should probably have done that for the sequential one :D)

Reply:
Agreed, that is slightly neater